trimmer.rb

```diff
@@ -0,0 +1,6 @@
+require_relative './basedecor'
+class TrimmerDecorator < BaseDecor
+  def trim
+    @nameable.correct_name.slice(0, 10)
+  end
```

Comment:
@mikethreels  the slice method already checks if the name is longer than 10 characters or not before using the slice in order to comply with the project requirements. because I'm slicing from index 0 to 10 if it's not there it gets ignored